src/librawspeed/decoders/IiqDecoder.cpp

```diff
@@ -151,13 +154,22 @@ RawImage IiqDecoder::decodeRawInternal() {
     case 0x10f:
       raw_data = bs.getSubView(data, len);
       break;
+    case 0x110:
+      correction_meta_data = bs.getSubStream(data);
```

Comment:
So now i have to re-review this all :/

Pretty sure this is
```c++
correction_meta_data = bs.getSubStream(data, len));
```

Reply:
For whatever reason the `len` value in this tag is garbage.  I just have to read the number of entries from the data

Reply:
:(